added user layout interaction support

limesTech opened the pull request:
## Summary of Changes
Closes #278 
Added a Callback function to divider to enable users to change viewport ratio between LogDetailsContainer and LogDetailsContainer

## Visual Inspection
<!-- Before approving, view the app in your browser and verify it doesn't have any of the below problems. -->

- [X] Conformance to Markdown styles: https://olog.esss.lu.se/Olog/help/CommonmarkCheatsheet
    - [X] ...when viewing a log entry
    - [X] ...when previewing HTML while writing a description
    - [X] ...when viewing a log entry in the group view
- [X]  Scroll is possible (elements don't overflow their container, and they are scrollable)
    - [X] ...search result list
    - [X] ...log entry group view list
    - [X] ...log entry single view
    - [X] ...create new log entry page
- [X] Overall layout fills full width and height of viewport
- [x] Pagination element doesn't overflow into other elements

maxfrederiksen-ess committed:
Fix websocket bugs

Commit:
added user layout interaction support

Commit:
pre-commit cleanup

maxfrederiksen-ess committed:
invalidate getLogGroup, fallback empty string description, New entry button

Commit:
update use re-resizable package instead

maxfrederiksen-ess committed:
Fix responiveness

maxfrederiksen-ess committed:
Rework logheader

maxfrederiksen-ess committed:
decrease zIndex expand all button

maxfrederiksen-ess committed:
IconButtons for log actions, history page arrow, cleanups

maxfrederiksen-ess committed:
2.5.0

maxfrederiksen-ess committed:
Collapse all, cleanup

Commit:
bug fixes

Commit:
cleanup code

maxfrederiksen-ess committed:
Edit entry warning

maxfrederiksen-ess committed:
Gallery touch-up

maxfrederiksen-ess committed:
Attachments and history touchup

maxfrederiksen-ess committed:
Rework v2

remove maxWidth owner

maxfrederiksen-ess commented:
Definitely better! However I do still have some comments:

1. There is a slight overflow on the entire screen, see screenshot.
<img width="1468" height="113" alt="Screenshot 2025-10-27 at 15 04 06" src="https://github.com/user-attachments/assets/7b5066d2-d664-4489-887b-2a1371130dab" />
2. I would like to still have a vertical divider like before but it's been removed.
3. Code is a bit messy and could most likely be simplified, however if you can tidy up the above 2 points I could refine the code if you want. However I will be gone on vacation most of November so will take a while.

maxfrederiksen-ess committed:
Preview on empty description crashes app

Commit:
aligned with repo standards

Commit:
fixed overflow

maxfrederiksen-ess committed:
Final touch ups

maxfrederiksen-ess committed:
line height

maxfrederiksen-ess committed:
2.5.1

maxfrederiksen-ess committed:
line height 100%

maxfrederiksen-ess committed:
CSSTUDIO-3533: timezone

maxfrederiksen-ess committed:
CSSTUDIO-3505: Download icon removed on edit page

maxfrederiksen-ess committed:
CSSTUDIO-3530: Disable edit submit if not edits have been made

maxfrederiksen-ess committed:
CSSTUDIO-3385: pre-commit info

maxfrederiksen-ess committed:
CSSTUDIO-3541: Dissallow submit for unmodified replys

You should not be allowed to submit a reply if nothing has changed from the original, since the original values will always be copied

maxfrederiksen-ess committed:
CSSTUDIO-2873: move edited to title, remove @ and #

maxfrederiksen-ess committed:
CSSTUDIO-2873: Added log id tooltip

maxfrederiksen-ess committed:
CSSTUDIO-3615: Don't copy attachments or description

dependabot[bot] committed:
Bump storybook from 9.1.6 to 9.1.17

Bumps [storybook](https://github.com/storybookjs/storybook/tree/HEAD/code/core) from 9.1.6 to 9.1.17.
- [Release notes](https://github.com/storybookjs/storybook/releases)
- [Changelog](https://github.com/storybookjs/storybook/blob/next/CHANGELOG.md)
- [Commits](https://github.com/storybookjs/storybook/commits/v9.1.17/code/core)

---
updated-dependencies:
- dependency-name: storybook
  dependency-version: 9.1.17
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>

maxfrederiksen-ess committed:
2.6.0

maxfrederiksen-ess committed:
sync package-lock

maxfrederiksen-ess committed:
sync package-lock

maxfrederiksen-ess committed:
CSSTUDIO-3632: Fix file upload extension bug

maxfrederiksen-ess committed:
CSSTUDIO-3633: use heic2any to convert HEIC files to JPEG

Added loader indicator for file conversion process and moved logic into reusable hook useFileUtils.

maxfrederiksen-ess committed:
package-lock.json sync

maxfrederiksen-ess committed:
package-lock.json sync

maxfrederiksen-ess committed:
sync lock file

Commit:
added user layout interaction support

Commit:
pre-commit cleanup

Commit:
fixed overflow

Commit:
cleanup code

Commit:
update use re-resizable package instead

Commit:
aligned with repo standards

Commit:
bug fixes

Commit:
Merge branch 'resize-user-interaction' of https://github.com/limesTech/olog-ui into resize-user-interaction
